configs/setup_golden_image.py

```diff
@@ -25,28 +25,29 @@
 from tmms.utils import utils as tmms_utils
 
 
-def customize_golden(golden_tar, build_dir):
+def customize_golden(manconfig, golden_tar, build_dir):
```

Comment:
I'm not sure about that manconfig. I think there is an overlap of config values between "manconfig" and "arg_values" bellow. Lets keep it as is for this merge request, but I'll re-review this later.

Reply:
manconfig has all the things from /etc/tmms to prevent hardcoding values (like "seedy").   Yes a further review is probably in order.